R/psprintf.R

```diff
@@ -0,0 +1,35 @@
+#' Paste together sprintf-style text and variable values
+#'
+#' Helper function to sprintf a bunch of key-value (string-variableVector) pairs,
+#' then paste them together with a good separator for contructing remake recipes.
+#'
+#' @param sep separator for the list of arguments, defaults to new line
```

Comment:
Add a line for: 
`#' @param \dots {explanation}` ?

Otherwise we'll get a check warning for an undocumented arg when we get in better shape w/ the package check stuff.

Reply:
Yup I can add a description

Reply:
For the description, how about "arguments that are either (1) character strings to insert directly into the output or (2) named arguments, where the name is an sprintf pattern and the value is a list or vector of variables to apply to that particular sprintf pattern. After any relevant sprint evaluation, all arguments will be pasted, with the specified separator, in order to create a single character output."

Reply:
Ah yeah that's much better